dark-facebook-messenger-theme.user.css

```diff
@@ -23,14 +23,35 @@
         color: #1a1a1a;
     }
 
-    ._4sp8, .clearfix._17pz {
+    /*Normal texts, originally black text on white background*/
+    /*._29_7 ._3058._s1-._6e6n : "... replied to you" / "You replied to..." boxes */
+    ._4sp8, .clearfix._17pz, ._3egs, ._3058, ._29_7 ._3058._s1-._6e6n {
```

Comment:
I think by adding this style to the incoming messages makes the "chat bubbles" blend too much into the background and it becomes harder to read. I prefer the incoming messages styles as they are.